src/components/Footer/UpperSection/index.js

```diff
@@ -31,19 +32,19 @@ import weiboIcon from "@/images/btn-weibo.svg"
 import useMedia from "@/hooks/useMedia"
 
 const SocialIcon = ({ last, src, alt, href }) => (
-  <a href={href} target="_blank" rel="noreferrer noopener">
+  <OutboundLink href={href} target="_blank" rel="noreferrer noopener">
     <img css={socialIcon(last)} src={src} alt={alt} />
-  </a>
+  </OutboundLink>
 )
 
 const OtherPages = props => (
   <p css={theme => [fontStyle(theme), otherPages]} {...props} />
 )
 
-const Page = ({ children, ...props }) => (
-  <Link css={pageAnchor} {...props}>
+const Page = ({ children, to, ...props }) => (
+  <OutboundLink css={pageAnchor} href={to} {...props}>
```

Comment:
This is not outbound link. This is internal pages.

Reply:
fixed.